addons/web_editor/models/ir_attachment.py

```diff
@@ -84,3 +84,36 @@ def _can_bypass_rights_on_media_dialog(self, **attachment_data):
         - Non admin user uploading an unsplash image (bypass binary/url check)
         """
         return False
+
+    def write(self, vals):
+        res = super().write(vals)
+        for attachment in self:
+            if attachment.mimetype in SUPPORTED_IMAGE_MIMETYPES:
```

Comment:
Why only for supported mimetypes? :thinking: Are not you supposed to remove the related image.data if the attachment is updated in such a way the mimetype is not supported anymore?

Reply:
The goal here was only for performance and was to only perform the `search()` if the attachment is linked to an image. The goal was not especially to check if the mimetype is supported but more to check if the mimetype is an image in general

Reply:
Ok :+1: